### PR TITLE
[RUIN-348] Document Home and formSections

### PR DESCRIPTION
**Pull request by 18chowdhary**

### CODE REVIEW

File: src/screens/formSections/NonMotoristSection.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+The NonMotoristSection is a class component that renders the tile for a nonmotorist section on the home screen.
+If a user clicks on the tile, they are navigated to the NonMotorist form. The component has two views: the regular view,
+which just displays an icon representing a nonmotorist, and the edit view, which displays a minus icon and allows the
```
Comment (KristinAoki) [inline src/screens/formSections/NonMotoristSection.js:4]: Change to “representing no motorist form”

File: src/screens/home/Home.js [hunk 2]
```diff
@@ -25,315 +35,357 @@ import { SvgXml } from "react-native-svg";
...
+                        rightMargin={4}
+                        text="Add Vehicle"
+                        onPress={() => this._addVehicleSection()}
+                        icon={
+                           <Icon color="white" name="local-taxi" size={50} />
+                        }
+                     />
+                  )}
+               </VStack>
+               {this.state.edit ? (
```
Comment (KristinAoki) [inline src/screens/home/Home.js:281]: Do we want to move this check to the non motorist section? It is a redundant check that is already done in the nonmotist section.

File: src/screens/home/Home.js [hunk 3]
```diff
@@ -25,315 +35,357 @@ import { SvgXml } from "react-native-svg";
...
-          ) : (
-            <>
-              {nonmotoristListArr.length ? (
-                <Section title="Non-motorists">
-                  <HStack flexWrap="wrap">{nonmotoristListArr}</HStack>
-                </Section>
-              ) : null}
-              <Section title="Crash Diagram">
-                {photo.image ? (
+      // render the right view
```
Comment (KristinAoki) [inline src/screens/home/Home.js:236]: What does this mean?

File: src/screens/formSections/VehicleSection.js [hunk 4]
```diff
@@ -1,3 +1,22 @@
+/*
+The VehicleSection is a class component that renders the tiles for a vehicle section on the home screen. This
+includes a tile for the vehicle, a tile for the driver and tiles for any passengers. If the user clicks on the
+tiles, they are navigated to the appropriate form. The component has two views: the regular view, which displays
+the icon buttons, and the edit view, which allows users to delete the vehicle or delete drivers or passengers
+inside the vehicle.
+
+const {navigation, vehicle, index, name, edit, passenger, roadID, questions } = this.props
```
Comment (KristinAoki) [inline src/screens/formSections/VehicleSection.js:8]: What is the point of this line?

File: src/screens/home/Home.js (hunk 2 above)
Comment (18chowdhary) [reply src/screens/home/Home.js:281]: It is slightly redundant but also a little complex. Currently, the nonmotorist section represents a single nonmotorist, so adding this button may screw up the rendering. I would prefer to keep the add nonmotorist button separate.

File: src/screens/home/Home.js (hunk 3 above)
Comment (18chowdhary) [reply src/screens/home/Home.js:236]: idk why this is here in the GitHub preview; it's supposed to be above the render function and just indicating that we are indicating either the edit view or the non-edit view dependent on the value of this.state.edit. Should I update the comment to clarify?

File: src/screens/formSections/VehicleSection.js (hunk 4 above)
Comment (18chowdhary) [reply src/screens/formSections/VehicleSection.js:8]: oops sorry that was for my reference when I was writing out the props

File: src/screens/home/Home.js (hunk 3 above)
Comment (KristinAoki) [reply src/screens/home/Home.js:236]: Yes please

**Review by KristinAoki — APPROVED** [review]

**Pull request merged**